### ISSUE TITLE
Verifying RDF file with invalid URI prints error but returns 0

### ISSUE THREAD

**Issue by timurphy** [spdx/tools#74]
Running Verify on an RDF file containing an invalid URI, such as [this file](https://gist.github.com/timurphy/25f0b72b2b93308f68b7#file-lgpl_sic_spdx-rdf), prints the below errors but exits with return code 0. As this is not a valid file, it should have a non-zero return code.

```
 WARN [main] (RDFDefaultErrorHandler.java:36) - file:////tmp/lgpl_spdx.rdf(line 21 column 137): {W107} Bad URI: <file:///srv/fossology/repository/report/SPDX2_lgpl_sic.txt_1456357178.rdf#LicenseRef-GPL-2.1[sic]> Code: 0/ILLEGAL_CHARACTER in FRAGMENT: The character violates the grammar rules for URIs/IRIs.
 WARN [main] (RDFDefaultErrorHandler.java:36) - file:////tmp/lgpl_spdx.rdf(line 76 column 149): {W107} Bad URI: <file:///srv/fossology/repository/report/SPDX2_lgpl_sic.txt_1456357178.rdf#LicenseRef-GPL-2.1[sic]> Code: 0/ILLEGAL_CHARACTER in FRAGMENT: The character violates the grammar rules for URIs/IRIs.
This SPDX Document is not valid due to:
	Invalid license id 'LicenseRef-GPL-2.1[sic]'.  Must start with 'LicenseRef-' and made up of the characters from the set 'a'-'z', 'A'-'Z', '0'-'9', '+', '_', '.', and '-'.
```

**Comment by timurphy** [comment]
The invalid portion is `LicenseRef-GPL-2.1[sic]` as square brackets are illegal characters

**Comment by goneall** [comment]
Fixed in version 2.1.1 - just pushed the changes to git.spdx.org - should show up on github within the hour.
